src/main/java/project/service/LibraryService.java

```diff
@@ -0,0 +1,108 @@
+package project.service;
+
+import project.dao.*;
+import project.entity.Book;
+import project.entity.Reader;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class LibraryService {
+    private LibraryDAO libraryDAO = new LibraryDAO();
+    private BookDAO bookDAO = new BookDAO();
+    private ReaderDAO readerDAO = new ReaderDAO();
+
+    public void showBooks() {
+        System.out.println(bookDAO.findAll());
```

Comment:
Предлагаю тебе весь этот класс перечитать и переделать на лямбды и новый синтаксис, где это возможно.
Например тут лучше будет
```suggestion
        bookDAO.findAll().forEach(System.out::println);
```

Reply:
Исправил, но пока без использования var